scripts/main.py

```diff
@@ -1,13 +1,24 @@
 import argparse
 import trailmap.parse_input as pi
 import trailmap.commodity_dictionary as cd
+import trailmap.acquisition_paths as ap
+# import trailmap.pathway_analysis as pa
+import networkx as nx
+from pprint import pprint
+import matplotlib.pyplot as plt
```

Comment:
is matplotlib actually required ? and sprint ?

Reply:
Another dependency that actually relates to the code I pulled out or this PR. Nothing currently in this PR uses it